ui/app/AppLayouts/Profile/Sections/MyProfileContainer.qml

```diff
@@ -53,76 +52,89 @@ Item {
             anchors.left: profileImg.right
             anchors.leftMargin: 8
             anchors.top: profileImg.top
-            font.weight: Font.Medium
-            font.pixelSize: 15
+            anchors.topMargin: 4
+            font.family: "Inter"
+            font.weight: Font.Bold
+            font.pixelSize: 20
         }
 
         Address {
             id: pubkeyText
             text: ensName !== "" ? username : pubkey
             anchors.bottom: profileImg.bottom
             anchors.left: profileName.left
+            anchors.bottomMargin: 4
             width: 200
             font.pixelSize: 15
         }
 
-        SVGImage {
-            id: qrCodeImage
-            source: "../../../img/qr-code-icon.svg"
-            width: 18
-            height: 18
-            anchors.right: parent.right
+        Rectangle {
```

Comment:
I think the StatusIconButton does pretty much the same thing but more simply

Reply:
thats true, ive used a StatusIconButton instead and was able to get the same results but with much shorter code! thanks!